lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -9,19 +9,45 @@
                     "name": "deposit",
                     "from_states": [],
```

Comment:
@revgum You may be right; the `from_state` may not be required here, and I don't want to break `find_deposit_action`. :)